pkg/etcd/etcd_test.go

```diff
@@ -0,0 +1,163 @@
+package etcd
+
+import (
+	"context"
+	"fmt"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	corev1 "k8s.io/api/core/v1"
+	policyv1 "k8s.io/api/policy/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+
+	commonLabels "github.com/medik8s/common/pkg/labels"
+)
+
+var guardLabel = map[string]string{"app": "guard"}
+
+var _ = Describe("Check etcd disruptions", func() {
+	var (
+		cl client.WithWatch
+	)
+
+	When("etcd PDB was not set", func() {
```

Comment:
what do you mean with "was not set"?

Reply:
was not created or when this PDB is missing.

Reply:
maybe "etcd PDB doesn't exist" would be better then?